CITATION.cff

```diff
@@ -0,0 +1,84 @@
+cff-version: "1.2.0"
+message: "If you use this template or the associated website, please cite it using these metadata."
```

Comment:
At the moment this asks people to use this citation for both template _and_ website. In #330 we discussed possibly making website separately citeable so we may want to change this.

Reply:
If we do then we could use the `preferred-citation` syntax https://github.com/astro-informatics/sleplet/blob/e3dec589a7f9803100c9aff6bdefa7ef76a6aa52/CITATION.cff#L10